src/iuse_actor.cpp

```diff
@@ -361,8 +361,24 @@ void consume_drug_iuse::load( JsonObject &obj )
 
 void consume_drug_iuse::info( const item&, std::vector<iteminfo>& dump ) const
 {
+    std::string vits;
+    for( const auto &v : vitamins ) {
+        // only display vitamins that we actually require
+        if( g->u.vitamin_rate( v.first ) > 0 ) {
+            if( !vits.empty() ) {
+                vits += ", ";
+            }
+            int lo = int( v.second.first  / ( DAYS( 1 ) / float( g->u.vitamin_rate( v.first ) ) ) * 100 );
+            int hi = int( v.second.second / ( DAYS( 1 ) / float( g->u.vitamin_rate( v.first ) ) ) * 100 );
```

Comment:
> The conditional above should exclude this?

Oops, my bad. Sorry.

>  that would be a good example of extreme premature optimization

Its not. Temps are useful to get rid of code duplication (I'd consider `g->u.vitamin_rate( v.first )` three times as a minor duplication, especially since they contain `g->u`) and clarify the intent via their names (off course, they're harmful when you can make code simpler without them).